src/fedora/27/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM fedora:27
+
+# Workaround to avoid NuGet restore timeout
+RUN dnf upgrade -y nss \
+    && dnf clean all
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
+# this does not include libraries that we need to compile different projects, we'd like
+# them in a different layer.
+RUN dnf install -y \
+        clang \
+        cmake \
+        lldb-devel \
+        llvm-devel \
+        make \
+        python \
+        which \
+    && dnf clean all
+
+# Install tools used by the VSO build automation.
+RUN dnf install -y \
+        git \
+        npm \
+        nodejs \
+        tar \
+        zip \
+    && dnf clean all
+
+RUN npm install -g azure-cli --unsafe-perm
+
+# Dependencies of CoreCLR and CoreFX.
+RUN dnf install -y \
+        glibc-locale-source \
+        libcurl-devel \
+        libgdiplus \
+        libicu-devel \
+        libunwind-devel \
+        libuuid-devel \
+        lttng-ust-devel \
+        uuid-devel \
+        compat-openssl10-devel \
```

Comment:
The standard Dockerfile format is to alphabetize the package lists in order to make it easier to read.  Is there a reason this can't get alphabetized?

Reply:
There is not. Updated.